source/common/conn_pool/conn_pool_base.cc

```diff
@@ -533,6 +539,13 @@ void ConnPoolImplBase::onConnectionEvent(ActiveClient& client, absl::string_view
       onUpstreamReady();
     }
     checkForIdleAndCloseIdleConnsIfDraining();
+    break;
+  }
+  case Network::ConnectionEvent::ConnectedZeroRtt: {
```

Comment:
is this the only thing we want to do ? don't we want to handle early data requests, or should that be a TODO?